cmd/statreceiver/parser.go

```diff
@@ -54,12 +54,6 @@ func (p *Parser) Packet(data []byte, ts time.Time) (err error) {
 	}
 
 	app, inst := string(appb), string(instb)
-	for _, filter := range p.filters {
-		if !filter.Filter(app, inst) {
-			return nil
```

Comment:
the function `NewParser` still takes `filters` as an argument. Should that be removed so it only accepts `dest MetricDest` now since it isn't using the filters anymore?

Reply:
> the function NewParser still takes filters as an argument.
whoops! you're right! oversight. the parser no longer needs filters of any kind

there's basically two levels of data - packets (groups of metrics) and metrics (just the key/values themselves). a parser turns a packet into metrics. because all of the filtering now operates on packets and metrics, the parser no longer needs to do it

Reply:
this is very helpful "a parser turns a packet into metrics", i couldn't figure it out just from reading the code.